src/Chapter2.hs

```diff
@@ -740,11 +764,14 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: (Eq a) => a -> [[a]] -> [[a]]
+contains _ []       = []
+contains c (x : xs) = if elem c x then x : rest else rest
+  where rest = contains c xs
```

Comment:
Great!

You could also use `filter` function, that does actually the same that you wrote in here 🙂 But it is great that you can easily implement standard functions, great job 👍🏼